components/src/core/Drawer/Drawer.tsx

```diff
@@ -284,11 +288,15 @@ const DrawerRenderer: React.ForwardRefRenderFunction<unknown, DrawerComponentPro
                 <div
                     style={{ flexDirection: 'column', flex: '1 1 0px', display: 'flex' }}
                     onMouseEnter={(): void => {
```

Comment:
Looks good, I would just change one thing...put the conditionals outside instead of inside the events. So we don't have a bunch of events firing off that aren't doing anything, i.e.:

```
onMouseEnter={ onOpenHover ? (): void => { hoverDelay = setTimeout(...) } : undefined }
```

Reply:
Change should be pushed out now.